mentoring/tests/integration/test_dashboard.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2014-2015 Harvard, edX & OpenCraft
```

Comment:
That's my preference as well, but I believe @antoviaque wants to use the file-headers style.